redhat/patches/0001-dockerfile.patch

```diff
@@ -1,22 +1,28 @@
 diff --git a/Dockerfile b/Dockerfile
-index 961d117..7d04f0c 100644
+index f2d39ac..4d33566 100644
 --- a/Dockerfile
 +++ b/Dockerfile
 @@ -13,7 +13,7 @@
  # See the License for the specific language governing permissions and
  # limitations under the License.
  
--FROM golang:1.21.1@sha256:cffaba795c36f07e372c7191b35ceaae114d74c31c3763d442982e3a4df3b39e AS builder
+-FROM golang:1.20.5@sha256:344193a70dc3588452ea39b4a1e465a8d3c91f788ae053f7ee168cebf18e0a50 AS builder
```

Comment:
Did you mean to change the image ref here and below?

Reply:
Yes I was having issues with the patch file, this patchfile will work for redhat-v1.2.2, but not release-next, cause the newest one uses 1.21 but were still on 1.20. I was wondering if it would be better to have two patch files? one for each release

Reply:
I figured since were building using redhat-v1.2.2 it would be better to prioritize it for now

Reply:
Actually now that I think of it this patch file should probably be in midstream-v1.2.2, and the one in main should be left alone? Am I right in saying this

Reply:
> Actually now that I think of it this patch file should probably be in `midstream-v1.2.2`, and the one in main should be left alone? Am I right in saying this

This patch file should include changes to the labels. If the patch doesn't apply, it's probably because upstream has changed the SHA. In fact, I think that's probably the issue. This automated patch file update should land first (https://github.com/securesign/rekor/pull/59) and then you can rebase this PR on top of that.

The patch file in midstream should not need SHA updates.